test/jmespath/jmespath.spec.js

```diff
@@ -75,35 +61,30 @@ function executeTestWithFields(desc, tst) {
   expect(result).toEqual(tst.result === undefined ? tst.error : tst.result);
 }
 
-test.each(toTestFmt(basic))('%s', executeTest);
+const suites = [
+  'basic', 'boolean',
+  'current', 'escape',
+  'filters', 'functions',
+  'identifiers', 'indices',
+  'literal', 'multiselect',
+  'pipe', 'slice',
+  'syntax', 'unicode',
+  'wildcard'
+];
+
+suites.forEach(suite => {
```

Comment:
I find it more readable in the command line and easy to identify which test has failed. I can revert this file, if you find it problematic.

Reply:
seems fine to me